yarn-project/end-to-end/src/e2e_public_cross_chain_messaging.test.ts

```diff
@@ -35,30 +42,32 @@ describe('e2e_public_cross_chain_messaging', () => {
   let inbox: any;
   let outbox: any;
 
+  beforeAll(async () => {
```

Comment:
nice. this should help reduce some flake.

Reply:
We should actually move all `setup`s to `beforeAll`. Otherwise, we get different behaviour when run locally vs when run connected to a running sandbox.